docker-compose.yml

```diff
@@ -82,14 +82,14 @@ services:
     # to it.
     command: >-
       sh -c '
-        cmd="daemon --upgrade"
+        cmd="daemon --consensus raft"
```

Comment:
Shouldn't we make this an environment variable?

Reply:
The consensus component choice, that is.

Reply:
mm no, other component-selection flags are not configurable via env-var.

Reply:
technically, for docker containers, it should be mandatory to pass `--consensus` when doing docker run. That should be possible for every tool that wants to start a docker container though (even if they also configure other things with env-vars).

Reply:
I'll deal with docker in other PR, I just wanted to maintain behaviour "as is" on this one.